aggregator_core/README.md

```diff
@@ -0,0 +1,18 @@
+# `aggregator_core`
+
+`aggregator_core` contains helper code for the `aggregator` crate. It mainly consists of data
+structures that are only pertinent to a DAP aggregator, and subroutines for talking to a PostgreSQL
+database.
+
+It is not published to crates.io and should not be depended on directly by users of Janus.
+
+## PostgreSQL Logs
+
+During tests, you can have PostgreSQL dump its logs to stdout by setting
+`JANUS_TEST_DUMP_POSTGRESQL_LOGS`. The test database is set to log all query plans. You should only
+run this option with one test at a time, otherwise you might not get all the logs.
```

Comment:
I suspect this is a testcontainers bug--if running multiple tests at at time, the output is incomplete, and perhaps interspersed with other containers. IMO it's not a big deal, since we'd only care to run this against one test at a time, so I don't care to spend too much time on it.

Reply:
Actually, we run multiple tests against the same container, to save on startup time, and separate by using different databases within the same process. Thus, database logs from queries in unrelated test aggregators will be commingled. It's also possible that we may shut down one database, and start another, if all the relevant Arcs go out of scope simultaneously. I wonder if there's some sort of race in the testcontainers library where deleting a container interrupts log forwarding, and if killing the container first, then deleting it once the log stream is done, would fix this.

Reply:
Good point about sharing a container (I had thought it was 1:1, since I didn't read the code closely, and because I think the phenomena you describe about Arcs going out of scope applies, showing as multiple postgres containers in `docker ps`).

[It looks like the log consumer task is spawned via `tokio::spawn`](https://github.com/testcontainers/testcontainers-rs/blob/main/testcontainers/src/core/containers/async_container.rs#L71), and the corresponding handle is never joined, so testcontainers never waits for the log stream to be drained before the process is exited. So I think a bunch of log lines get queued up, but never get serviced because the process ends too quickly.

Indeed, if I tack a `sleep()` at the end of an affected test, I see a much more plausible volume of output.